app/controllers/concerns/application_shared.rb

```diff
@@ -15,4 +15,35 @@ def set_timezone
     # Reset timezone for the next thread
     Time.zone = default_timezone
   end
-end
\ No newline at end of file
+
+  private
+
+  # Get the allowed parameters from the Apipie documentation in the API controllers.
+  def safe_params
+    apipie_params = Apipie.app.get_method_description(controller_name, action_name).to_json[:params]
```

Comment:
When we first thought of strong params, we thought to separate api params and web ui params.
This unifies them?